addons/quickmount/ACE_Settings.hpp

```diff
@@ -36,6 +36,6 @@ class ACE_Settings {
         description = CSTRING(PriorityDescription);
         isClientSettable = 1;
         force = 0;
-        values[] = {"Driver", "Gunner", "Commander", "Passenger"};
```

Comment:
Yeah, but I only found 2:
- Driver: `str_a3_cfgvehicles_c_driver_1_random_base_f0`
- Commander: `str_a3_cfgvehicles_highcommand_moduledescription_anybrain_0`

Might be better to just translate them but put them in `common` component so they can be used in more places.